log.p8

```diff
@@ -0,0 +1,48 @@
+pico-8 cartridge // http://www.pico-8.com
+version 18
+__lua__
+-- pico8lib logging library
+-- by mindfilleter
+
+-- Depends on pico8lib/strings.p8
+
+--- Log the passed in object to STDOUT
+-- The passed in object will be converted to a string and then logged with a
+-- timestamp.
+-- @param level A level prefix to prepend to the message
+-- @param any The object to log
+-- @return nil
+local function log (level, any)
+   local message = stat(90)
+   message = message .. "-" .. stat(91)
+   message = message .. "-" .. stat(92)
+   message = message .. " " .. stat(93)
+   message = message .. ":" .. stat(94)
+   message = message .. ":" .. stat(95)
+   printh(message .. level .. " - " .. tostr(any))
+end
```

Comment:
I think it would save tokens and time to do `printh(stat(90) .. "-" .. stat(91) ..`...

Reply:
I'd also like to see an option to log in UTC instead of local time